*** custom_script_loader.cpp    2021-01-23 20:52:16.478510735 +0000
--- custom_script_loader.cpp.patched    2021-01-23 20:55:59.055167411 +0000
***************
*** 16,24 ****
--- 16,26 ----
   */

  // This is where scripts' loading functions should be declared:
+ void AddSC_solocraft();

  // The name of this function should match:
  // void Add${NameOfDirectory}Scripts()
  void AddCustomScripts()
  {
+       AddSC_solocraft();
  }
